### PR TITLE
Updated test files and regenerated interfaces.go

### PR DESCRIPTION
**Pull request by everjing**
Update test files:token_retriever_test,token_exchanger_test,authorizer_test and regenerate interfaces.go in mock_authorizer package

### CODE REVIEW

File: pkg/authorizer/token_retriever.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func NewWorkloadIdentityTokenRetriever() *WorkloadIdentityTokenRetriever {
 }
 
 // AcquireARMToken acquires the managed identity ARM access token
```
Comment (niontive) [inline pkg/authorizer/token_retriever.go]: Make sure to update the comments

File: pkg/authorizer/interfaces.go [hunk 2]
```diff
@@ -18,3 +18,13 @@ type Interface interface {
 type ACRTokenExchanger interface {
 	ExchangeACRAccessToken(armToken types.AccessToken, tenantID, acrFQDN string) (types.AccessToken, error)
 }
+
+// MIARMTokenRetriever is the interface to retrieve an ARM access token via managed identity.
+type MIARMTokenRetriever interface {
+	MIAcquireARMToken(clientID, resourceID string) (types.AccessToken, error)
```
Comment (niontive) [inline pkg/authorizer/interfaces.go]: Could we just call this method `AcquireARMToken`? It's ok for both the MI and WI token retrievers to have the same method name for their interfaces

File: pkg/authorizer/interfaces.go [hunk 3]
```diff
@@ -18,3 +18,13 @@ type Interface interface {
 type ACRTokenExchanger interface {
 	ExchangeACRAccessToken(armToken types.AccessToken, tenantID, acrFQDN string) (types.AccessToken, error)
 }
+
+// MIARMTokenRetriever is the interface to retrieve an ARM access token via managed identity.
+type MIARMTokenRetriever interface {
```
Comment (niontive) [inline pkg/authorizer/interfaces.go]: Per Weinong's suggestion, avoid shortened names (ex: spell out Managed Identity).

File: pkg/authorizer/interfaces.go [hunk 4]
```diff
@@ -18,3 +18,13 @@ type Interface interface {
 type ACRTokenExchanger interface {
 	ExchangeACRAccessToken(armToken types.AccessToken, tenantID, acrFQDN string) (types.AccessToken, error)
 }
+
+// MIARMTokenRetriever is the interface to retrieve an ARM access token via managed identity.
```
Comment (niontive) [inline pkg/authorizer/interfaces.go]: Nit: comment says `MIARMtokenRetriever` instead of `ManagedIdentityARMTokenRetriieever`

**Review by niontive — APPROVED** [review]

File: pkg/authorizer/interfaces.go (hunk 4 above)
Comment (everjing) [reply pkg/authorizer/interfaces.go]: Resolved in the most recent commit. Please check it out. Thanks!

**Pull request merged**